components/base/wheeled/wheeled_base.go

```diff
@@ -1,11 +1,17 @@
 // Package wheeled implements some bases, like a wheeled base.
 package wheeled
 
+/*	This driver package enables control of a wheeled base, supporting
```

Comment:
can you look at my open PR for IMU and grab the note from there at the top of the file?